open-api/rest-catalog-open-api.yaml

```diff
@@ -1441,6 +1435,34 @@ components:
   #############################
   responses:
 
+    RESTCatalogConfigResponse:
+      description:
+        Configuration from the server consists of two sets of key/value pairs.
```

Comment:
I thought that you needed 2 newlines between paragraphs, and that `*` didn't work for bulleted lists.